webapp/src/components/progress_bars/TrainingBar.vue

```diff
@@ -4,7 +4,7 @@
       v-if="taskTitle !== undefined && displayTitle"
       class="flex flex-wrap font-disco text-3xl justify-center"
     >
-      <span class="text-disco-blue">{{ taskTitle }}</span>
+      <span class="text-disco-blue dark:text-disco-light-cyan">{{ taskTitle }}</span>
```

Comment:
Nice! Can you do the something similar when displaying DISCOllaboratives at the Task Selection step?